HTTPShortcuts/app/src/main/kotlin/ch/rmy/android/http_shortcuts/activities/settings/polling/models/ShortcutListItemId.kt

```diff
@@ -0,0 +1,5 @@
+package ch.rmy.android.http_shortcuts.activities.settings.polling.models
```

Comment:
This file does not have to be duplicated. But I'll need to find a good package for the other file